### ISSUE TITLE
Change wording of "Block" in notifications to "Mute"

### ISSUE THREAD

**Issue by dariusk** [hometown-fork/hometown#1289]
![image](https://user-images.githubusercontent.com/266454/212733621-650b3fdd-8b04-412c-986b-090ddb57f100.png)

The above settings do not operate as "block", they operate as "mute" (silently discarding things while not notifying the other party), and they should be listed as such.

**Comment by mcclure** [comment]
See also: https://github.com/mastodon/mastodon/issues/11882

And be aware of this, which you may or may not consider significant:

> A third, more minor issue: If a DM has been hidden by the checkbox, if the checkbox is later un-checked, the DM from the checkboxed period stays hidden... This is a surprising behavior.

(This might actually be how the notifications tab behaves in the case of a normal mute, now that I think about it.)

**Comment by dariusk** [comment]
@mcclure Yeah that is how the notifications tab behaves for mutes, too. It is a little bit different from "mute" on other social network sites but it's consistent with "mute" in other areas of Mastodon.

**Comment by mark-franz** [comment]
Indeed I would prefer to have this function working as "real" block, not just mute. However for sake of clarity, as long it's just a mute, I would also support to change the wording to "mute". 
(Background: today I was searching for the "block" function since I was recommended to do so due to increasing spam on the instance. I was searching for quite a while and was surprised to find it under "notifications".)
